Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -448,17 +476,35 @@ public void Wzhzhzh(EntityUid uid, MicrowaveComponent component, EntityUid? user
 
                 AddComp<ActivelyMicrowavedComponent>(item);
 
-                var metaData = MetaData(item); //this simply begs for cooking refactor
-                if (metaData.EntityPrototype == null)
+                string? solidID = null;
+                int amountToAdd = 1;
```

Comment:
remove the amountToAdd stuff as that should be determined by the recipe

Reply:
The amountToAdd is just used for adding to the buffer. For the non-stacked entities, it's 1, but for a stacked entity, it's however big the stack is. I don't know how this is related to the recipe.